### DISCUSSION TITLE
Drawing nodes behind links

### DISCUSSION THREAD

**Discussion by MerlinB** [vasturiano/force-graph#351] [Q&A] [answered]
Hi, thank you for this library. I have a graph where the user can select a node to highlight the nodes links and neighbours. However, I could not find a way to draw the highlighted links above the background nodes, even using `linkCanvasObjectMode` and `nodeCanvasObjectMode`. Is there a way to do this?

<img width="606" alt="Screenshot 2024-05-19 at 5 51 41 PM" src="https://github.com/vasturiano/force-graph/assets/7633719/e63370bd-ebc9-41b4-a976-71c60c904432">

**Comment by vasturiano** [comment, marked as answer]
@MerlinB thanks for reaching out.

Unfortunately this is not currently supported. Regardless of whether you have default or custom node/links the drawing order is always links first and nodes after. That seemed to be by far the most common expected sequence, that's why it was done like that. But I can imagine a reasoning around your use case too. Well, you can somewhat alleviate what you're describing by making the background nodes semi-transparent. That way you'll be able to at least see-through to the links.

**Comment by MerlinB** [comment]
I see, thank you for the explanation.
